extensions/amp-analytics/0.1/vendors.js

```diff
@@ -454,6 +454,59 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'mparticle': {
+    'vars': {
+      'apiKey': '',
+      'eventName': '',
+      'eventType': 'Unknown',
+      'eventAttributes_Keys': [],
```

Comment:
Developers are not usually going to see this. A better place for this would be in the amp-analytics.md or in https://docs.mparticle.com/?javascript#amp/. 

Adding the variables here increases size of the extension for everyone who uses amp-analytics(regardless of whether mparticle is being used or not).